source/WebConfig/webconfig_internal.c

```diff
@@ -670,13 +744,13 @@ int processJsonDocument(char *jsonData, int *retStatus, char **docVersion)
 	if(parseStatus ==1)
 	{
 		WebcfgDebug("Request:> Type : %d\n",reqObj->reqType);
-		WebConfigLog("Request:> ParamCount = %zu\n",reqObj->u.setReq->paramCnt);
+		WebcfgDebug("Request:> ParamCount = %zu\n",reqObj->u.setReq->paramCnt);
```

Comment:
Better to enable log for these as we are currently not storing config response in DB.

Reply:
Incorporated.